float64[] wifiMean
float64[] wifiStd
